xml/System/Half.xml

```diff
@@ -59,8 +59,8 @@
         <Parameter Name="other" Type="System.Half" />
       </Parameters>
       <Docs>
-        <param name="other">To be added.</param>
-        <summary>Compares this object to another object, returning an integer that indicates the relationship.</summary>
+        <param name="other">A Half-precision floating-point number to compare.</param>
+        <summary>Compares this instance to a specified Half-precision floating-point number and returns an integer that indicates whether the value of this instance is less than, equal to, or greater than the value of the specified Half-precision floating-point number.</summary>
```

Comment:
should these be `half-precision`, given that single/double use `single-precision` and `double-precision` (lowercase)?